java/java.lsp.server/vscode/README.md

```diff
@@ -66,6 +71,12 @@ Language Server __Java 8+ ...__ launch configuration supports debugging and runn
     * __Attach to Shared Memory__ is available on Windows in addtion to above mentioned _Attach..._
 
 Default launch configurations provided by Language Server can modified in `launch.json` file using intellisense for available options.
+
+### Run Configurations panel
+Program arguments, VM options, evironment,... can be set in Run Configuration panel, part of Explorer instead of in `launch.json`.
```

Comment:
A (pandemic) typo "e-vironment" ;-)

Overall I am not sure what's the message of this paragraph? Should it be something like:

_Tired of editing `launch.json` configuration file manually? Use Run Configuration Panel instead! It simplifies setting of program arguments, VM options, environment variables & co. The panel is sufficient for all typical use-cases Java programmer faces. Only advanced, expert scenarios may require touching of `launch.json` (which still takes precedence) - stay cool - use Run configuration Panel!_

Reply:
Used with less superlatives to keep more documentation like style.